sdk/python/lib/pulumi/invoke.py

```diff
@@ -131,10 +135,137 @@ def merge(
             if source.plugin_download_url is None
             else source.plugin_download_url
         )
-        dest.version = dest.version if source.version is None else source.version
+        dest.version = (
+            dest.version if (source.version in [None, ""]) else source.version
+        )
+
+        return dest
+
+
+class InvokeOutputOptions(InvokeOptions):
+    """
+    InvokeOutputOptions is a bag of options that control the behavior of a call to runtime.invoke_output.
+    """
+
+    depends_on: Optional["Input[Union[Sequence[Input[Resource]], Resource]]"]
+    """
+    If provided, declares that the currently-constructing resource depends on the given resources.
```

Comment:
Might want to tweak this to talk about "currently-executing invoke" or similar, maybe?

Reply:
sorry, bad copy-pasta